src/CommandList.java

```diff
@@ -0,0 +1,36 @@
+
+import java.util.HashMap;
+import java.util.Map;
+public class CommandList {
+    HashMap<String,Command> testCommand;
+    private String add = "add";
+    private String getAllUsers = "getAllUsers";
+    private String getUserForId = "getUserForId";
+
+    AddCommand addCommand = new AddCommand();
+    GetAllUsersCommand getAllUsersCommand = new GetAllUsersCommand();
+    GetUserForIdCommand getUserForIdCommand = new GetUserForIdCommand();
+
+    public CommandList() {
+        testCommand = new HashMap<>();
+        testCommand.put(add,addCommand);
+        testCommand.put(getAllUsers,getAllUsersCommand);
+        testCommand.put(getUserForId,getUserForIdCommand);;
+    }
+
+    public Map<String,Command> initilization(){
+        return testCommand;
+    }
+
+//    public CommandList (AddCommand addCommand, GetAllUsersCommand getAllUsersCommand, GetUserForIdCommand getUserForIdCommand) {
```

Comment:
it is not recommended to keep commented code. If you don't need this method, just remove. Never comment